hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/token/delegation/AbstractDelegationTokenSecretManager.java

```diff
@@ -34,6 +38,8 @@
 import org.apache.hadoop.classification.InterfaceAudience;
 import org.apache.hadoop.classification.InterfaceStability;
 import org.apache.hadoop.io.Text;
+import static org.apache.hadoop.metrics2.util.Metrics2Util.NameValuePair;
```

Comment:
is it necessary to import static?

Reply:
Inigo told me once about it and I think it can have the usage of member variables easier and cleaner.

Reply:
If the method/class is very obvious like assertTrue(), it usually makes sense to do a static import.
In this case, I guess is fine either way.

Reply:
Not always the case but I'd say this is fine.